tests/testthat/test-edit.R

```diff
@@ -1,37 +1,49 @@
 context("edit")
 
-expect_user_file <- function(...) {
-  expect_true(file.exists(scoped_path("user", ...)))
+expect_r_file <- function(...) {
+  expect_true(file_exists(scoped_path_r("user", ...)))
 }
 
-expect_user_git_file <- function(...) {
-  expect_true(file.exists(scoped_git_path("user", ...)))
+expect_fs_file <- function(...) {
+  expect_true(file_exists(scoped_path_fs("user", ...)))
 }
 
-expect_project_file <- function(...) expect_true(file.exists(proj_path(...)))
+expect_project_file <- function(...) expect_true(file_exists(proj_path(...)))
 
 ## testing edit_XXX("user") only on travis and appveyor, because I don't want to
 ## risk creating user-level files de novo for an actual user, which would
 ## obligate me to some nerve-wracking clean up
 
+test_that("edit_r_XXX() and edit_git_XXX() have default scope", {
+  ## run these manually if you already have these files or are happy to
+  ## have them or delete them
+  skip_if_not_ci()
+
+  capture_output(expect_error_free(edit_r_profile()))
+  capture_output(expect_error_free(edit_r_environ()))
+  capture_output(expect_error_free(edit_r_makevars()))
+  capture_output(expect_error_free(edit_git_config()))
+  capture_output(expect_error_free(edit_git_ignore()))
+})
+
```

Comment:
These functions previously didn't work if called with no input 🙈 Needed to call `match.arg()` before passing `scope` onto `switch()`.